sysapps/device_capabilities/cpu_info_provider_win.cc

```diff
@@ -4,14 +4,27 @@
 
 #include "xwalk/sysapps/device_capabilities/cpu_info_provider.h"
 
+#include <pdh.h>
+
 #include "base/logging.h"
 
 namespace xwalk {
 namespace sysapps {
 
+static PDH_HQUERY cpuQuery;
```

Comment:
Do we really needed anonymous? `cpuQuery` is not declared in header. BTW, `cpuQeury` -> `cpu_query`